### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by J3rkYs** [NiekNijland/CSGO-Translator#13] [Q&A]
I saw reddit post about the translator,but when i instaled it, it says "Logfile not found". I have no idea what it means. Can someone pls help me.
![Snímek obrazovky 2021-05-04 135907](https://user-images.githubusercontent.com/83647443/117000152-ed5ed900-ace0-11eb-9c94-cff8dd10550d.png)

**Comment by NiekNijland** [comment]
@J3rkYs 

This error means that it can't find the file to which the console output is logged.
Did you follow the setup steps in the readme and set the 2 launch options in steam?

This could be caused by 3 things:

1. You didn't set the required launch options for CS:GO in steam.
2. You didn't start CS:GO yet, so the logfile has not been created yet.
3. You have set a wrong CS:GO install location in the CSGO-Translator options, so the translator is looking in the wrong place.

Please let me know if you need more help after checking these problems.

**Reply by J3rkYs** [reply]
Hello NiekNijland

I have set up the 2 launch options
![Snímek obrazovky 2021-05-04 194704](https://user-images.githubusercontent.com/83647443/117046923-8d345b00-ad11-11eb-8593-06e2fedb6708.png)

 I have started CSGO, but the issue is still there

I have double checked the instal location and it was wrong, but it didnt work even after I changed it
![Snímek obrazovky 2021-05-04 195234](https://user-images.githubusercontent.com/83647443/117048156-f9638e80-ad12-11eb-8368-a5adc5c4a774.png)
![Snímek obrazovky 2021-05-04 195822](https://user-images.githubusercontent.com/83647443/117048281-18fab700-ad13-11eb-9a7f-55227b9c23fb.png)

Anyway thank you for your help

**Reply by NiekNijland** [reply]
@J3rkYs Sorry for this very late reply. but in your screenshot the path is still in wrong. You have types Ofensive instead of Offensive.
